packages/react-liveness/src/components/FaceLivenessDetector/service/utils/CustomWebSocketFetchHandler.ts

```diff
@@ -0,0 +1,267 @@
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
```

Comment:
This file is mostly copied from - https://github.com/aws/aws-sdk-js-v3/blob/main/packages/middleware-websocket/src/websocket-fetch-handler.ts#L176

The file was not typed so added a bunch of typescript linting ignores

Reply:
Get why this was done but it kind of negates the effort we have gone in to regarding type safety and linting in the rest of the React packages 😢 If we absolutely need to do this think there should be an explanation above so future maintainers understand this was an exception and should not be used as precedent

Reply:
+1 And can we have a followup task to work through the linting errors and remove the ignores?

Reply:
Created a follow up task and added a comment